integration

```diff
@@ -1518,11 +1518,12 @@
   "zbigniewmotyka/home-assistant-solplanet",
   "zeronounours/HA-custom-component-energy-meter",
   "zhbjsh/homeassistant-ssh",
+  "zhheo/ha_honghui_climate",
```

Comment:
This is using non-English in commits, logs, and comments, so it is not intended for global consumption.
As this is only relevant for users in a specific country, the released `hacs.json` file needs a country key.
<https://hacs.xyz/docs/publish/include>